x-pack/filebeat/module/azure/activitylogs/test/activitylogs.log

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
I'm wondering, shouldn't this be a single line raw JSON? as the pipeline is doing the decoding part